### ISSUE TITLE
MCSG Maker Gamemaster

### ISSUE THREAD

**Issue by arsenalfcgunners** [MCGamerNetwork/Issues#870]
If a mod is hosting a game and goes into gamemaster within the lobby and then goes back out they don't get the nether star back.  Even if they relog.  I can't seem to find a way to get it back.  Additionally, it would be cool if they don't lose the nether star in the first place when they go into gamemaster.  When hosting MCG events with the community the staff member who is running the event may want to be in GM when hosting rather than a participant.

**Comment by CAmadeusA** [comment]
Is this still an issue?

**Comment by arsenalfcgunners** [comment]
Yes, at least on the stable servers it is an issue.  Not sure if it has changed since the last beta release.
